draft-ietf-quic-transport.md

```diff
@@ -4634,10 +4628,14 @@ preferred_address (0x000d):
 +                                                               +
 |                                                               |
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+| CID Length (8)|
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                      Connection ID (*)                      ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
```

Comment:
What's the rationale for reordering the contents of this TP payload in this PR?

Reply:
This came from https://github.com/quicwg/base-drafts/issues/2944 since they were both cosmetic.

Reply:
Based on feedback from the room, I've reverted that change.